data/scenarios/Tutorials/give.yaml

```
@@ -3,32 +3,59 @@ name: Give
 description: |
   Learn how to give items to other robots.
 objectives:
-  - goal:
+  - id: give_bit
+    teaser: Give base bit
+    goal:
       - |
         Robots can transfer items between them using the `give`
         command. This command takes two arguments: the target
         robot, and the name of the item to give.  The target robot
         is often the special `base` robot, but could be any robot.
-      - In this scenario, you will need to collect the items scattered around your `base`, by `build`ing one or more robots to collect them and then `give` them to you.
+      - |
+        In this scenario, you will need to collect the items scattered
+        around your `base`, by `build`ing one or more robots to collect
+        them and then `give` them to you.
+      - |
+        This is a training simulation, so the items will reappear after a while.
+      - |
+        Start with `build`ing a robot to `grab` the `bit (0)`{=entity} east of the base.
+    condition: |
+      as base { has "bit (0)" }
+  - id: give_rest
+    teaser: Fetch the rest
+    goal:
+      - |
+        Good job. Now send robots to grab the rest!
+      - |
+        Where will you find the item names, so you can `give base "item name"`?
+        That's the neat part, you won't!
+      - |
+        You will get the name from the `grab` command result:
```

```suggestion
        You will get the name from the `grab` command result, for example:
```
Just to make it clear that you are not necessarily supposed to run this exact code.